src/projectmembers/hooks/useSanitizedProjectMembers.ts

```diff
@@ -14,30 +14,31 @@ interface ProjectMemberBase {
 }
 
 export function useSanitizedProjectMembers<T extends ProjectMemberBase>(members: T[]): T[] {
-  const [sanitizedMembers, setSanitizedMembers] = useState<T[]>([])
+  return members
+  // const [sanitizedMembers, setSanitizedMembers] = useState<T[]>([])
 
-  useEffect(() => {
-    const updatedMembers = members.map((member) => {
-      if (member.deleted) {
-        if (member.name) {
-          // Deleted team member: anonymize the name
-          return { ...member, name: "Anonymous" }
-        } else if (member.name === null && member.users.length > 0) {
-          // Deleted individual contributor: sanitize user data
-          const updatedUsers = member.users.map((user) => ({
-            ...user,
-            username: "Anonymous",
-            firstName: user.firstName !== undefined ? "" : user.firstName,
-            lastName: user.lastName !== undefined ? "" : user.lastName,
-          }))
-          return { ...member, users: updatedUsers }
-        }
-      }
-      return member
-    })
+  // useEffect(() => {
+  //   const updatedMembers = members.map((member) => {
+  //     if (member.deleted) {
+  //       if (member.name) {
+  //         // Deleted team member: anonymize the name
+  //         return { ...member, name: "Anonymous" }
+  //       } else if (member.name === null && member.users.length > 0) {
+  //         // Deleted individual contributor: sanitize user data
+  //         const updatedUsers = member.users.map((user) => ({
+  //           ...user,
+  //           username: "Anonymous",
+  //           firstName: user.firstName !== undefined ? "" : user.firstName,
+  //           lastName: user.lastName !== undefined ? "" : user.lastName,
+  //         }))
+  //         return { ...member, users: updatedUsers }
+  //       }
+  //     }
+  //     return member
+  //   })
 
-    setSanitizedMembers(updatedMembers)
-  }, [members])
+  //   setSanitizedMembers(updatedMembers)
+  // }, [members])
 
-  return sanitizedMembers
+  // return sanitizedMembers
```

Comment:
Are these comments necessary? Good to delete if not needed anymore.

Reply:
You are right. The whole function is obsolete now since we do the anonymization in the queries themselves. I will remove it.